packages/graalvm/src/main/kotlin/elide/runtime/node/fs/NodeFilesystem.kt

```diff
@@ -336,7 +336,7 @@ internal abstract class FilesystemBase(
   // Obtain current context, if any, and then execute in the background; once execution completes, the context is again
   // entered, and execution continues.
   protected inline fun <reified T> withExec(noinline op: () -> T): JsPromise<T> = try {
-    exec.spawn<T> { op() }
+    exec.of { op() }
```

Comment:
i believe this actually files a task against the executor? rather than simply allocating, so it should probably be called `spawn<T>` or `exec<T>` or something to that effect

Reply:
Good point, the rename was caused by the changes to the `JsPromise` API, because the two methods (`spawn` and `of`) were the same, so in the rename `of` won I guess.